HelpSource/Classes/SequenceableCollection.schelp

```diff
@@ -889,6 +889,137 @@ copymethod:: SimpleNumber -atanh
 copymethod:: SimpleNumber -owensT
 
 
+subsection:: Testing
+
+method:: ==
+
+Overridden by the need to check for equality of each element with its integer index as independent of identity. See link::Classes/Object#Equality and Identity::.
+
+Discussion::
+code::
+~array_a = [1, 2]; ~array_b = [1, 2] // Evaluate this code first!
```

Comment:
Please avoid direct address of the reader, especially no exclamation marks !!

Reply:
Also it may be more practical to have an example with comments, instead of splitting one over several parts. Then it is clear that the top is needed.

Reply:
I wrote this for newbies. They need to be exposed to the code and the evaluated result. If I wrote this part within `code::` and `::`, many newbies would have difficulty understanding it. For expertise, we do not need such explanations.

Anyway, I changed this part a bit.